sdk/storage/storage-blob/src/Pipeline.ts

```diff
@@ -210,6 +211,7 @@ export function newPipeline(
   // The credential's policy factory must appear close to the wire so it can sign any
   // changes made by other factories (like UniqueRequestIDPolicyFactory)
   const factories: RequestPolicyFactory[] = [
+    new DfsPolicyFactory(),
```

Comment:
What about normal blob, will it's url be replaced to dfs's one?

Reply:
nO

---
In reply to: [318949843](https://github.com/Azure/azure-sdk-for-js/pull/4874#discussion_r318949843) [](ancestors = 318949843)